packages/mars/src/options/chain/model.ts

```diff
@@ -0,0 +1,13 @@
+export type Chain = {
+  chainId: number
+  chainName: string
+  getPublicRPC: () => string
+  getInfuraRPC: (infuraApiKey: string) => string
+  getAlchemyRPC: (alchemyApiKey: string) => string
+  getBlockExplorerContractAddress: (contractAddress: string) => string
+  getContractVerifierApi: () => string
```

Comment:
One thing I don't like about `getContractVerifierApi` is that it secretly assumes it's an Etherscan API.